ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKDownloadDelegate.swift

```diff
@@ -18,7 +18,8 @@ extension BrowserViewController: WKDownloadDelegate {
   ) async -> URL? {
 
     if let httpResponse = response as? HTTPURLResponse {
-      if httpResponse.mimeType != MIMEType.passbook {
+      if ![MIMEType.passbook, MIMEType.passbookBundle].contains(httpResponse.mimeType?.lowercased())
```

Comment:
Is it actually possible for the mime-type stored in `URLResponse` to be not lowercase for these mimtypes?

Reply:
The HTML spec says it's case-insensitive but handling of it depends on the developer. So if I send back `TEXT/HTML` I'm not sure if it'll be normalized to `text/html` when received. I'm not sure if iOS will do it.
So I lower-case it anyway.

Reply:
It's seemingly normalized in Swift: https://github.com/swiftlang/swift-corelibs-foundation/blob/c466923dcaeeb058fc570f0e0cadbb04afb2b5df/Sources/FoundationNetworking/URLResponse.swift#L217